packages/open-ui/src/components/Navigation/Navigation.module.scss

```diff
@@ -1,4 +1,4 @@
-@import 'styles/index.scss';
+@import '../../styles/index.scss';
```

Comment:
@remotealex we are providing the PATH_SASS in the .env file, ryt ?

Reply:
Rollup doesnt recognise the `PATH_SASS` so i removed it for now!